docs/api-testing/project-dashboard.md

```diff
@@ -1,43 +1,65 @@
+<img src={useBaseUrl('img/api-fortress/2021/02/sessionDetails.png')} alt="Test Report Primer Image" width="300" />
+
+#### General Details
+Test name, test status (pass or fail), project.
+<img src={useBaseUrl('img/api-fortress/2021/02/generalDetails.png')} alt="Test Report Primer Image" width="200" />
+
+#### Request Component
+Shows the specific component that you tested. To view granular details about requests and headers, click **See Stack Details**.
+<img src={useBaseUrl('img/api-fortress/2021/02/requestComponent.png')} alt="Test Report Primer Image" width="650" />
+
+#### Assertions
+Gray dot indicates a pass, red dot indicates a fail, and yellow dot indicates a warning.
+<img src={useBaseUrl('img/api-fortress/2021/02/assertions.png')} alt="Test Report Primer Image" width="500" />
+
 
 
 ## Test Logs
 
 To view Test logs:
 
-1. Log in to Sauce Labs, then click **API Testing** > **Get Started**.<br/><img src={useBaseUrl('img/api-fortress/2021/09/landingPage.png')} alt="API Testing landing page" width="500" />
+1. Log in to Sauce Labs, then click **API Testing**.
 2. Click on any Project, then click the **Dashboard** tab.<br/><img src={useBaseUrl('img/api-fortress/2021/02/dash-1.2.png')} alt="Dashboard Pic 1.2" width="300" />
-3. From inside the **Dashboard**, select **Logs**.<br/><img src={useBaseUrl('img/api-fortress/2021/02/dashboardLogs.png')} alt="Dashboard Logs"/>
+3. From **Dashboard**, select **Logs**.<br/><img src={useBaseUrl('img/api-fortress/2021/02/dashboardLogs.png')} alt="Dashboard Logs"/>
 4. Set desired filters to find the test(s) you're looking for, then click **Apply**. Available filters are: date range, tags, event type, and mode (Monitoring (Scheduled) Tests or On-Demand Tests).<br/><img src={useBaseUrl('img/api-fortress/2021/02/dashboardFilters.png')} alt="Dashboard Logs Filters"/>
 
-### Test Reports
-
-If you hover your mouse over a line item, then click the kebab icon (three dots), you'll see the option to go to the [Test Outcome Report](/api-testing/test-reports/).<br/><img src={useBaseUrl('img/api-fortress/2021/02/dashboardReport.png')} alt="Dashboard Report" width="500"/>
+To view the [Test Outcome Report](#test-outcome-reports/), hover your mouse over a line item, then click the kebab icon (three dots):<br/><img src={useBaseUrl('img/api-fortress/2021/02/dashboardReport.png')} alt="Dashboard Report" width="500"/>
 
 
 ## Test Metrics
 
-To view Test performance metrics (latency and fetch) within a Project:
+To view Test performance metrics (latency and fetch):
 
 1. Follow steps 1 and 2 from [Test Logs](#test-logs).
 2. From inside the **Dashboard**, select **Metrics**.<br/><img src={useBaseUrl('img/api-fortress/2021/02/metrics.png')} alt="Dashboard Metrics"/>
 3. Set the start and end date filters, if desired.<br/><img src={useBaseUrl('img/api-fortress/2021/02/metricsFilters.png')} alt="Dashboard Metrics Filters"/>
```

Comment:
@kimsauce we have added "footprint" and "build id" fields and the "all metrics" dropdown that allows user to filter only failures